openupgrade_framework/readme/CONFIGURE.rst

```diff
@@ -1,4 +1,4 @@
-* call your odoo instance with the option ``--load=web,openupgrade_framework``
+* call your odoo instance with the option ``--load=base,web,openupgrade_framework``
```

Comment:
Why adding base ?

Reply:
If we don't include it, /jsonrpc route doesn't work. Then, If I use for example odoorpc to do the analysis, it will be impossible. Remember that the default of load is `base,web`, so in any case, you should respect that and add modules, but not taking out.